langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -206,6 +289,14 @@ public isolated function getTarget(ProcessingInstruction x) returns string = @ja
 
 # Returns the content of a processing instruction or comment item.
 #
+# ```ballerina
+# xml:ProcessingInstruction p = xml `<?webpage ballerina.io?>`;
+# person.getContent() ⇒ ballerina.io
+# 
+# xml:Comment comment = xml `<!--Example comment-->`;
+# comment.getContent() ⇒ Example comment
```

Comment:
```suggestion
# xml:Comment comment = xml `<!--Employees by department-->`;
# comment.getContent() ⇒ Employees by department
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/9dc46b7c36c28b92b9ce360ed71621b3a6eff8c0